### ISSUE TITLE
GCLoader not being found when Swiss is booted from Pico

### ISSUE THREAD

**Issue by nikeymikey** [OffBroadway/flippyboot-ipl#27]
Running cubeboot from a picoboot install as ipl.dol

On booting the animation plays, Swiss boots and then a popup appears saying "Initialising DVD Drive" that is then replaced with "No DVD Drive found".  GCLoader is not detected and not available in Swiss device menu. 

If cubeboot (ipl.dol) file is removed and replaced with standard picoboot ipl.dol then Swiss loads and the GCLoader is detected straight away. 
Have tried flashing the pico with cubeboot.uf2, same results occur. 

Have tried removing all boot delays in the .ini along with different versions of swiss. Currently using r1476 (latest).

…

**Steps to Reproduce**
Have both GCLoader and Picoboot installed. Boot cubeboot from pico. 

…

**Expected behavior**
> GCLoader should be detected and GCLoader SD card files should show in Swiss. 

…

**Comment by zombie343** [comment]
Does this mean GCLoader and cubeboot don't work together? 😂

**Comment by nikeymikey** [comment]
Yes. If you boot from cubeboot then Swiss does not recognise the GCLoader
or it’s SD card at all.

**Comment by zombie343** [comment]
Damn it. I really hoping this would work. What SD boot card options exist without GCLoader? SD2SP2? I heard the SD2MemoryCard option works but is slow. :(

**Comment by nikeymikey** [comment]
Cubeboot cannot be launched from the GCloader afaik btw, if thats what you were planning to do. 
You will need a Picoboot to launch cubeboot or an old school IPL modchip. SD2SP2 is where I store all my homebrew files, all my games are on the GCLoader SD card. The Picoboot launches Swiss form the SD2SP2 card and then I can boot games from GCLoader.
